### PR TITLE
Bug 1172465 - border-radius should be applied to all buttons

### PR DESCRIPTION
**Pull request by ayushagrawal288**
Corrected errors specified after last merge
Screenshots attached:
https://www.dropbox.com/s/koeiphcieqtjzh6/screenshot1.jpg?dl=0
https://www.dropbox.com/s/1517ssf4i6s7tde/screenshot2.png?dl=0

### CODE REVIEW

File: src/media/css/newsletter.styl [hunk 1]
```diff
@@ -206,6 +206,7 @@ $arrow-size = 10px;
     .newsletter-cancel-button {
         width: auto;
         margin: 0;
+        border-radius: 5px;
```
Comment (spasovski) [inline src/media/css/newsletter.styl]: We tend to add CSS rules in alphabetic order for consistency.

Comment (ayushagrawal288) [inline src/media/css/newsletter.styl]: how can i correct it ??

Comment (spasovski) [inline src/media/css/newsletter.styl]: `border-radius: $button-border-radius;
margin: 0;
width: auto;`

**Comment by ngokevin** [comment]
i think the bug is fixed? anyways, let us know when this bug is updated. thanks! sorry for the delay

**Pull request closed without merging**